### PR TITLE
add case format to case import

### PR DESCRIPTION
**Pull request by jamal-khey**

### CODE REVIEW

File: src/test/java/org/gridsuite/study/server/StudyServiceTest.java [hunk 1]
```diff
@@ -153,6 +154,8 @@ public void testCheckNetworkExistenceReturnsNotContent() throws Exception {
 
         mockMvc.perform(head("/v1/studies/{studyUuid}/network", studyUuid)
                 .header(USER_ID_HEADER, userId))
+//                .param(CASE_FORMAT_PARAM, "UCTE"))
```
Comment (flomillot) [inline src/test/java/org/gridsuite/study/server/StudyServiceTest.java:157]: Forgot to clean ?

File: src/test/java/org/gridsuite/study/server/utils/TestUtils.java [hunk 2]
```diff
@@ -165,6 +165,8 @@ public static void assertServerRequestsEmptyThenShutdown(MockWebServer server) t
     public static void assertWiremockServerRequestsEmptyThenShutdown(WireMockServer wireMockServer) throws UncheckedInterruptedException, IOException {
         try {
             wireMockServer.checkForUnmatchedRequests(); // requests no matched ? (it returns an exception if a request was not matched by wireMock, but does not complain if it was not verified by 'verify')
+            var req = wireMockServer.findAll(WireMock.anyRequestedFor(WireMock.anyUrl())); // requests no verified ?
```
Comment (flomillot) [inline src/test/java/org/gridsuite/study/server/utils/TestUtils.java:168]: Forgot to clean also ?
Anyway you can remove the duplicated line and reuse `req` below

**Comment by Meklo** [comment]
I'm getting this error when creating a study, any idea where it comes from ? 
```
java.lang.ClassCastException: class com.rabbitmq.client.impl.LongStringHelper$ByteArrayLongString cannot be cast to class java.lang.String (com.rabbitmq.client.impl.LongStringHelper$ByteArrayLongString is in unnamed module of loader 'app'; java.lang.String is in module java.base of loader 'bootstrap')
	at org.hibernate.type.descriptor.java.StringJavaType.unwrap(StringJavaType.java:27) ~[hibernate-core-6.2.6.Final.jar:6.2.6.Final]
	at org.hibernate.type.descriptor.jdbc.VarcharJdbcType$1.doBind(VarcharJdbcType.java:108) ~[hibernate-core-6.2.6.Final.jar:6.2.6.Final]
	at org.hibernate.type.descriptor.jdbc.BasicBinder.bind(BasicBinder.java:61) ~[hibernate-core-6.2.6.Final.jar:6.2.6.Final]
	at org.hibernate.engine.jdbc.mutation.internal.JdbcValueBindingsImpl.lambda$beforeStatement$0(JdbcValueBindingsImpl.java:87) ~[hibernate-core-6.2.6.Final.jar:6.2.6.Final]
[...]
at org.gridsuite.study.server.service.NetworkModificationTreeService.isNodeNameExists(NetworkModificationTreeService.java:539) ~[classes/:na]
```

File: src/main/java/org/gridsuite/study/server/service/CaseService.java [hunk 3]
```diff
@@ -75,10 +75,11 @@ public void deleteCase(UUID caseUuid) {
         }
     }
 
-    public UUID duplicateCase(UUID caseUuid, Boolean withExpiration) {
+    public UUID duplicateCase(UUID caseUuid, Boolean withExpiration, String caseFormat) {
         String path = UriComponentsBuilder.fromPath(DELIMITER + CASE_API_VERSION + "/cases")
                 .queryParam("duplicateFrom", caseUuid)
                 .queryParam("withExpiration", withExpiration)
+                .queryParam("caseFormat", caseFormat)
```
Comment (Meklo) [inline src/main/java/org/gridsuite/study/server/service/CaseService.java:82]: Wouldn't that mean that a powsybl case server PR is also needed ?

File: src/test/java/org/gridsuite/study/server/StudyServiceTest.java (hunk 1 above)
Comment (jamal-khey) [reply src/test/java/org/gridsuite/study/server/StudyServiceTest.java]: done

File: src/main/java/org/gridsuite/study/server/service/CaseService.java (hunk 3 above)
Comment (jamal-khey) [reply src/main/java/org/gridsuite/study/server/service/CaseService.java:82]: i forget to clean, done now.
about the error in the logs , i have no idea, i don't have this error
can you please see if you still have it after the changes i made ?

Comment (jamal-khey) [reply src/main/java/org/gridsuite/study/server/service/CaseService.java]: nice remark, i cleaned the code and updated the tests

Comment (jamal-khey) [reply src/main/java/org/gridsuite/study/server/service/CaseService.java]: i can't reproduce the error log you have. does not seem to be related to the changes in this PR
can you please check again with the current version of the code?

**Review by flomillot — APPROVED** [review]
Code OK

**Review by anistouri — APPROVED** [review]

**Review by Meklo — APPROVED** [review]

**Pull request merged**